recipe/meta.yaml

```
@@ -443,6 +461,7 @@
 }
 %}
 {% set version = cudavars[major_minor]["version"] %}
+{% set major_version = cudavars[major_minor]["major_version"] %}
```

Instead of adding the key above can we just parse the `version` string here to extract the major version?

---

Agreed! 
I guess this should work:

```suggestion
{% set major_version = version.split('.')[0]|int %}
```

---

thanks, I had the same thought this weekend. Guess I made my life to easy...